mlir/lib/Target/Cpp/TranslateToCpp.cpp

```diff
@@ -1412,7 +1473,7 @@ LogicalResult CppEmitter::emitGlobalVariable(GlobalOp op) {
     os << "const ";
 
   if (failed(emitVariableDeclaration(op->getLoc(), op.getType(),
-                                     op.getSymName()))) {
+                                     op.getSymName(), false))) {
```

Comment:
Shall globals also be allowed to bear references? e.g. we declare a global and then issue another one which is a reference to the former.

Reply:
yes, we should also allow it for globals; they are essentially the same as emitc.variable but in a large scope

Reply:
Allowed, we can't verify them now as there's no way to define a global with another global.